src/api/lib/obsapi/markdown_renderer.rb

```diff
@@ -41,7 +41,9 @@ def link(link, title, content)
 
     def block_code(code, language)
       language ||= :plaintext
-      CodeRay.scan(code, language).div(css: :class)
+      CodeRay.scan(code, language).div
```

Comment:
I'm assuming you are removing the `css: :class` because it's the default?

Reply:
yeah